timelock-server/src/test/java/com/palantir/atlasdb/timelock/lock/ExclusiveLockTests.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.timelock.lock;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Future;
+
+import org.junit.Test;
+
+public class ExclusiveLockTests {
+
+    private static final UUID REQUEST_1 = UUID.randomUUID();
```

Comment:
Technically yeah, this makes the tests non-deterministic, but I think that this is fine as the only behavior we care about from these UUIDs is that they are unique.